optuna/study/study.py

```diff
@@ -125,12 +125,13 @@ def best_value(self) -> float:
     def best_trial(self) -> FrozenTrial:
         """Return the best trial in the study.
 
-        Returns:
-            A :class:`~optuna.trial.FrozenTrial` object of the best trial.
+        .. note::
+            A study must have exactly one direction.
+            If your study is multi-objective,,
+            use :attr:`~optuna.study.Study.best_trials` instead.
 
-        Raises:
-            :exc:`RuntimeError`:
-                If the study has more than one direction.
+        Returns:
+            This feature can only be used for single-objective optimization.
```

Comment:
The different line is updated. Please fix appropriately.

Reply:
[966391c](https://github.com/optuna/optuna/pull/3429/commits/966391cb045c3e98f846ad85919f36fb2ca025b6)